armi/operators/snapshots.py

```diff
@@ -24,6 +24,10 @@ class OperatorSnapshots(operatorMPI.OperatorMPI):
 
     These may add CR worth curves, rx coefficients, transient runs etc at these snapshots.
     This operator can be run as a restart, adding new physics to a previous run.
+
+    .. impl:: Save extra data to be saved from a run, at specified time nodes.
+        :id: I_ARMI_SNAPSHOT1
+        :implements: R_ARMI_SNAPSHOT
```

Comment:
I think this should either 1) go on the snapshotInterface.py file or 2) go on operator.py::Operator::snapshotRequest. 

Dumping/saving data (used via the snapshotInterface) and using the SnapshotOperator are two different things. See #1472 for more thoughts.

Reply:
This is ALSO in `snapshotInterface.py`.

Does that help?

Reply:
Okay, I removed it from here.